src/components/dialog/Dialog.js

```diff
@@ -47,10 +68,11 @@ class Dialog extends PureComponent {
     return (
       <DialogBase className={classNames} {...restProps}>
         {title && this.getHeader()}
-        <Box className={theme['dialog-body']} padding={4}>
+        <Box className={theme['dialog-body']} padding={4} ref={this.bodyRef}>
```

Comment:
@nickwaelkens Apparently for Lennert createRef works 🙃

Reply:
The better front-ender 🙈

Reply:
Teach us

Reply:
Eat lots of green vegetables and stay hydrated.

Reply:
Only if you start bets on shaving it